Library/Styles/ProjectStyles.swift

```diff
@@ -3,6 +3,50 @@ import Prelude
 import Prelude_UIKit
 import UIKit
 
+public enum ProjectCampaignExperimentType {
```

Comment:
I don't think we need this interstitial type. The button style can just be a function in the view model:

```swift
public func projectCampaignButtonStyleForVariant(
  _ variant: OptimizelyExperiment.Variant
) -> ProjectCampaignButtonStyleType {
  switch variant {
  case .control:
    return .controlReadMoreButton
  case .variant1, .variant2:
    return .experimentalReadMoreButton
  }
}
```
And then the signals in `ProjectPamphletMainCellViewModel` can just be:
```swift
let projectCampaignExperimentType = self.awakeFromNibProperty.signal
  .map { projectCampaignExperiment() }
  .skipNil()
self.readMoreButtonStyle = projectCampaignExperimentType.map(projectCampaignButtonStyleForVariant)
self.spacerViewHidden = projectCampaignExperimentType.map { $0 != .control }
self.blurbAndReadMoreStackViewSpacing = projectCampaignExperimentType.map { $0 == .control ? 0 : 4 }
  .map(Styles.grid)
```